.gitignore

```diff
@@ -3,3 +3,4 @@
 .tmp
 *.pyc
 __pycache__
+.vscode
```

Comment:
Need a line feed in every file. Add this setting in VSCode.

```json
"files.insertFinalNewline": true
```

Reply:
Or you can just add the editorconfig extension and it would do it for you.

Reply:
This is still not fixed. https://stackoverflow.com/a/54270768/2352587